meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,311 @@
+<br/>
+
+---
+## ⚡ Supercharge your Meilisearch experience
+
+Say goodbye to server deployment and manual updates with [Meilisearch Cloud](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=engine&utm_medium=meilisearch). No credit card required.
+
+---  
+<br/>
+
+# New features and improvements 🔥
+
+## Vector Search — Experimental
+
+You can now use Meilisearch as a vector store. Meilisearch allows you to add vector embeddings generated by third-party software, and use embeddings when searching with the `/search` or `/multi-search` routes.
+
+This experimental feature can be enabled via the HTTP API using v1.3.0's new `/experimental-features` endpoint.
+
+```bash
+curl \
+  -X PATCH 'http://localhost:7700/experimental-features/' \
+  -H 'Content-Type: application/json'  \
+  --data-binary '{
+    "vectorStore": true
+  }'
+```
+
+### Sending Vectorized Documents
+
+For the first iteration of this feature you must compute the vectors separately. Once that is done, include them in your documents using the `_vectors` field. Finally, send the documents with vector data to your instance. A single document may contain multiple vectors.
+
+⚠️ Vector size must be the same across all documents in a dataset. If vector sizes are inconsistent, Meilisearch will return an error during document addition.
+
+```bash
+curl -X POST -H 'content-type: application/json' \
+  'localhost:7700/indexes/songs/documents' \
+  --data-binary '[
+      { "id": 0, "_vectors": [0, 0.8, -0.2], "title": "Across The Universe" },
+      { "id": 1, "_vectors": [1, -0.2, 0], "title": "All Things Must Pass" },
+      { "id": 2, "_vectors": [[0.5, 3, 1], [-0.2, 4, 6]], "title": "And Your Bird Can Sing" }
+  ]'
+```
+
+### Query Meilisearch using Vectors
+
+Use the new `vector` search parameter with the `/search` and `/multi-search` to search for documents with the nearest vector. You must compute the vector query with a third-party tool.
+
+```bash
+curl -X POST -H 'content-type: application/json' \
+'localhost:7700/indexes/songs/search' \
+--data-binary '{ "vector": [0, 1, 2] }'
+```
+
+### Similarity score
+
+When you use vector search, returned documents include a `_semanticScore` field.
+
+```json
+{
+  "hits": [
+    { "id": 0, "_vectors": [0, 0.8, -0.2], "title": "Across The Universe", "_semanticScore": 0.6754 },
+    { "id": 1, "_vectors": [1, -0.2, 0], "title": "All Things Must Pass", "_semanticScore": 0.7546 },
+    { "id": 2, "_vectors": [[0.5, 3, 1], [-0.2, 4, 6]], "title": "And Your Bird Can Sing", "_semanticScore": 0.78 }
+  ],
+  "query": "",
+  "vector": [0, 1, 2],
+  "processingTimeMs": 0,
+  "limit": 20,
+  "offset": 0,
+  "estimatedTotalHits": 2
+}
+```
+
+
+🗣️ This feature is experimental and we need your help to improve it! Share your thoughts and feedback on [this GitHub discussion](https://github.com/meilisearch/product/discussions/677).
+
+⚠️ Experimental features are unstable and may be incompatible between Meilisearch versions.
```

Comment:
@guimachiavelli

Sorry, you can feel I put too much time on these details, but for me it's not.

I'm not comfortable with the word "unstable" to define our experimental features here. We named the exp features `experimental` and not `unstable` on purpose to avoid the bad advertisement it could bring. "unstable" can refer to bugs and a lack of "performance" stability, which is not here. It's only about API change.
The previous sentence was maybe less pretty, but more accurate from my POV.

<img width="944" alt="Capture d’écran 2023-07-03 à 15 55 38" src="https://github.com/meilisearch/engine-team/assets/20380692/14f9f0a9-e274-4af9-9776-1e273f74b827">


Pinging also @dureuill who was in the discussion

Reply:
Ok, easy enough to fix. My proposal:

```suggestion
⚠️ Experimental features may be incompatible between Meilisearch versions.
```

Reply:
I've updated here and the other reference as well. Should be all good now